pkgs/applications/editors/vim/plugins/aliases.nix

```diff
@@ -86,6 +86,7 @@ mapAliases (with prev; {
   indent-blankline-nvim-lua = indent-blankline-nvim; # backwards compat, added 2021-07-05
   ipython             = vim-ipython;
   latex-live-preview  = vim-latex-live-preview;
+  magma-nvim-goose    = magma-nvim; # fork has been archived in favor of the original project
```

Comment:
suggestion: add a deprecation warning.

Reply:
Good idea, I moved it to `deprecated.json`:
```
error: magma-nvim-goose was renamed to magma-nvim on 2024-10-28. Please update to magma-nvim.
```